.circleci/config.yml

```diff
@@ -2,258 +2,18 @@ version: 2.1
 orbs:
   release-management: salesforce/npm-release-management@4
 
-jobs:
-  registry-check:
-    description: Checks registry against metadataCoverageReport
```

Comment:
Can we pull the descriptions over to the workflow files? Bummer if not

Reply:
yep

Reply:
oops, turns out not.  I'll do it with # comments